extensions/wikia/Flags/controllers/FlagsController.class.php

```diff
@@ -146,6 +138,57 @@ public function getFlagsForPage() {
 		$this->setResponseData( $flagsForPage );
 	}
 
+	public function postFlagsEditForm() {
+		$this->processRequest();
+
+		if ( !isset( $this->params['pageId'] ) ) {
+			return null;
+		}
+
+		/**
+		 * Get the current status to compare
+		 */
+		$currentFlags = $this->getAllFlagTypes( $this->params['wikiId'], $this->params['pageId'] );
+
+		$helper = new Helper();
+		$flagsToChange = $helper->compareDataAndGetFlagsToChange( $currentFlags, $this->params );
+
+		$this->model = new Flag();
```

Comment:
Yep, I think that at this point we can easily say that storing a model as the Controller's property is pointless. Thanks!